Solution1/Solution1.Tests/BL/Extensions/ForecastWeatherDTOExtensionsTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using BusinessLayer.DTOs;
+using BusinessLayer.Extensions;
+using KellermanSoftware.CompareNetObjects;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Weather.Tests.BL.Extensions
+{
+    public class ForecastWeatherDTOExtensionsTests
+    {
+        [Fact]
+        public void GetMultiStringRepresentation_GetMultiStringRepresentationFromWeatherDTO_Success()
+        {
+            // Arrange
+            
+            // Act
+            var result = GetForecastWeatherDTO().FillCommentByTemp().GetMultiStringRepresentation();
```

Comment:
no. in Act only ONE method should be called - the method which you test: GetMultiStringRepresentation;
All test data should be prepared in advance - in Arrange section, not using any methods.
Now you are testing TWO methods

Reply:
+